scout/constants/igv_tracks.py

```diff
@@ -11,8 +11,11 @@
 HG38ALIAS_URL = "https://igv.org/genomes/data/hg38/hg38_alias.tab"
 HG38CYTOBAND_URL = "https://igv-genepattern-org.s3.amazonaws.com/genomes/hg38/cytoBandIdeo.txt.gz"
 
-HG38GENES_URL = "https://s3.amazonaws.com/igv.org.genomes/hg38/refGene.txt.gz"
-HG38GENES_FORMAT = "refgene"
+HG38GENES_URL = "https://s3.amazonaws.com/igv.org.genomes/hg38/Homo_sapiens.GRCh38.94.chr.gff3.gz"
+HG38GENES_INDEX_URL = (
+    "https://s3.amazonaws.com/igv.org.genomes/hg38/Homo_sapiens.GRCh38.94.chr.gff3.gz.csi"
+)
+HG38GENES_FORMAT = "gff3"
```

Comment:
This is working, but track loading is quite slow

<img width="943" alt="image" src="https://github.com/user-attachments/assets/953fd56d-21e3-4230-aacb-1194563f12e5" />